gcore/src/utils.rs

```diff
@@ -113,3 +149,40 @@ pub mod ext {
         unsafe { gsys::gr_oom_panic() }
     }
 }
+
+/// Add a debug message to the log.
+///
+/// Same as `gstd::debug!()` but uses a stack-allocated buffer.
```

Comment:
it can use stack, but not always:
https://github.com/gear-tech/gear/blob/b9511183910d6b1ff6a018a5cadb72dc784b4fee/stack-buffer/src/lib.rs#L100-L104

Reply:
https://github.com/gear-tech/gear/pull/4351/files#diff-2203746c3be74ee3be9d80b919b9ed85f5fe98f45a953f96f7cb92eae852d6aeR92

Reply:
ok, add same note for macro. not only for `fn stack_debug()`